demo/src/FlatParametersTab.js

```diff
@@ -118,6 +118,7 @@ export const FlatParametersTab = () => {
                     paramsAsArray={EXAMPLE_PARAMETERS}
                     initValues={currentParameters1}
                     variant={'standard'}
+                    showSeparator={true}
```

Comment:
showSeparator is enough. No need for **showSeparator = {true}**

Reply:
done